.github/workflows/validate.yml

```diff
@@ -110,6 +110,14 @@ jobs:
       docker-services: cassandra init-keyspace dev-chain-fast deploy-network-subgraphs-fastchain
       run-entry-point: true
       command: npm run test-integration
+  broker-e2e-fast:
```

Comment:
The streamr-1.0 branch now uses fastchain, so we don't need this if you merge latest from that branch

Reply:
Removed